tests/command/cursor-tailable-001.phpt

```diff
@@ -27,8 +27,6 @@ $command = new MongoDB\Driver\Command([
     'pipeline' => $pipeline,
     'cursor' => ['batchSize' => 0],
 ], [
-    'tailable' => true,
-    'awaitData' => true,
```

Comment:
This were added in https://github.com/mongodb/mongo-php-driver/pull/673/files#diff-b83b98cb9d11fe3a7c88272acfb1bb87R30.